services/events/migrations/versions/5dadfc5db16d_.py

```diff
@@ -1,16 +1,16 @@
 """empty message
 
-Revision ID: ed60e04bb208
+Revision ID: 5dadfc5db16d
```

Comment:
The `migration` file has been squashed and replaces the existing migration file; An audit of the schema will be carried out in the near future; to avoid a complex migration history all migrations will be merged until a final schema is agreed.